core-bundle/tests/InsertTag/Resolver/DateInsertTagTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Contao\CoreBundle\Tests\InsertTag\Resolver;
+
+use Contao\CoreBundle\InsertTag\ResolvedInsertTag;
+use Contao\CoreBundle\InsertTag\ResolvedParameters;
+use Contao\CoreBundle\InsertTag\Resolver\DateInsertTag;
+use PHPUnit\Framework\TestCase;
+
+class DateInsertTagTest extends TestCase
+{
+    /**
+     * @dataProvider expiresAtProvider
+     */
+    public function testExpiresAt(string $format, \DateTimeImmutable|null $expectedExpiresAt): void
+    {
+        $insertTag = new DateInsertTag();
+        $result = $insertTag(new ResolvedInsertTag('date', new ResolvedParameters([$format]), []));
+        $this->assertSame($expectedExpiresAt?->getTimestamp(), $result->getExpiresAt()?->getTimestamp());
+    }
+
+    public function expiresAtProvider(): \Generator
+    {
+        yield 'Null when not cacheable' => [
+            'H:i:s',
```

Comment:
```suggestion
            'd.m.Y H:i:s',
```
This would currently fail.